zenoh/src/net/runtime/orchestrator.rs

```diff
@@ -603,14 +612,15 @@ impl Runtime {
         // Must set to nonblocking according to the doc of tokio
         // https://docs.rs/tokio/latest/tokio/net/struct.UdpSocket.html#notes
         socket.set_nonblocking(true)?;
+        socket.set_multicast_ttl_v4(multicast_ttl)?;
```

Comment:
The documentation of [`UdpSocket`](https://doc.rust-lang.org/std/net/struct.UdpSocket.html#method.set_multicast_ttl_v4) states that "this might not have any effect on IPv6 sockets.". Perhaps we can add something like:

```rust
if sockaddr.is_ipv6() && multicast_ttl > 1 {
  tracing::warn!("UDP Multicast TTL has been set to a value greater than 1 on a socket bound to an IPv6 address. This might not have the desired effect");
}

```